dspace-cris/api/src/main/java/org/dspace/app/cris/util/UnsubscribeEPersionUtils.java

```diff
@@ -0,0 +1,162 @@
+package org.dspace.app.cris.util;
+
+import java.util.List;
+import java.util.Locale;
+
+import org.apache.commons.lang.StringUtils;
+import org.apache.log4j.Logger;
+import org.dspace.app.cris.model.ACrisObject;
+import org.dspace.app.cris.service.ApplicationService;
+import org.dspace.app.cris.service.CrisSubscribeService;
+import org.dspace.app.cris.statistics.StatSubscriptionViewBean;
+import org.dspace.app.cris.statistics.service.StatSubscribeService;
+import org.dspace.content.Collection;
+import org.dspace.content.Community;
+import org.dspace.core.ConfigurationManager;
+import org.dspace.core.Constants;
+import org.dspace.core.Context;
+import org.dspace.core.Email;
+import org.dspace.core.I18nUtil;
+import org.dspace.eperson.EPerson;
+import org.dspace.eperson.Subscribe;
+import org.dspace.utils.DSpace;
+
+public class UnsubscribeEPersionUtils {
+	/** log4j logger */
+	private static Logger log = Logger.getLogger(UnsubscribeEPersionUtils.class);
+	
+	/**
+	 * Unsubscribe an eperson from all the notification.
+	 * 
+	 * @param context The context
+	 * @param email The mail related to an ePerson
+	 * @throws Exception
+	 */
+	public static void process(
+			Context context, String email) throws Exception {
+				
+		EPerson eperson = EPerson.findByEmail(context, email);
+		if (eperson == null) {
+			throw new Exception("The email " + email + " is not related to any ePerson.");
+		}
+		
+		log.debug("Using email template: unsubscriptions");
+		Email recoverEmail = Email.getEmail(I18nUtil.getEmailFilename(Locale.getDefault(), "unsubscriptions"));
+		String recipient = ConfigurationManager.getProperty("alert.recipient");
+		String notify = ConfigurationManager.getProperty("registration.notify");
+		if (StringUtils.isNotBlank(recipient)) {
```

Comment:
the script must accept an argument to disable the notification and if the notification has been disabled no exception should be thrown